### DISCUSSION TITLE
Coding on an iPad

### DISCUSSION THREAD

**Discussion by Jules1111** [coder/code-server#3593] [Q&A] [answered]
Please can you help me find out how to get VS Code on my iPad please- I don’t have a MacBook or laptop.  I saw a post online by Joe  Previte About this and despite his very helpful and Step by step messages telling me how to do this I keep getting so confused and have Not been able to do it yet Does anyone have any tips please ? Thank you so much!  I want to learn coding. Jules

**Comment by jsjoeio** [comment]
Hey Jules! Let me see if I can put together a video walkthrough for you.

**Reply by jsjoeio** [reply]
There's a tiny bug with our Railway Guide. Once https://github.com/cdr/deploy-code-server/pull/36 is approved tomorrow, I'll make a video walkthrough! Thanks for your patience!

**Reply by jsjoeio** [reply]
Sorry, I have a task that is higher priority but I will get to this soon!

**Reply by Jules1111** [reply]
Hey that’s ok I really really appreciate your help! Thank you so much!

**Comment by Jules1111** [comment]
Ah thank you so much! May I ask is my email address public on github? I didn’t realise..I’d like to change the email address I use there can I do that ? Jules 

Sent from Yahoo Mail for iPhone


On Friday, June 11, 2021, 12:20 am, Joe Previte ***@***.***> wrote:




There's a tiny bug with our Railway Guide. Once cdr/deploy-code-server#36 is approved tomorrow, I'll make a video walkthrough! Thanks for your patience!

**Reply by jsjoeio** [reply]
There is some information [here](https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address)

**Comment by Jules1111** [comment]
Ah thank you ! 
Jules

**Comment by jsjoeio** [comment, marked as answer]
Here's a video which walks you through deploying code-server to Railway and using on an iPad!

https://www.youtube.com/watch?v=IWfkkaY5fXs

**Comment by Jules1111** [comment]
Oh wow thank you !

**Reply by jsjoeio** [reply]
Sure thing! I hope it helps! Feel free to ask follow-up questions here if you have any

**Comment by Jules1111** [comment]
Wow this is great thank you ! 
Which iPad storage do you think would be best to do this on ? 
Thank you so much! 
Jules

**Reply by jsjoeio** [reply]
Great question! Any of the storage options should be fine. This doesn't store anything locally. Everything (coding files, etc) are all inside your Railway environment. 

Well actually, your environment is not persisted so basically what you'll probably do is:
- set up code-server & Railway env
- clone a GitHub/GitLab repo
- work on your project
- push to remote so changes are backed up with version control

This is because your Railway environment might restart, reboot, etc. so it's not guaranteed your changes will be saved. I hope that makes sense!

**Comment by Jules1111** [comment]
Thank you I’m  doing my best to understand:) 
I will try this this evening on my old 2013 iPad just to see if it works although it’s quite slow - I will probably need to upgrade it but iPads are so much better in my opinion! Thank you for all your help Jules 


Sent from Yahoo Mail for iPhone


On Wednesday, June 16, 2021, 11:17 pm, Joe Previte ***@***.***> wrote:




Great question! Any of the storage options should be fine. This doesn't store anything locally. Everything (coding files, etc) are all inside your Railway environment.

Well actually, your environment is not persisted so basically what you'll probably do is:
   
   - set up code-server & Railway env
   - clone a GitHub/GitLab repo
   - work on your project
   - push to remote so changes are backed up with version control

This is because your Railway environment might restart, reboot, etc. so it's not guaranteed your changes will be saved. I hope that makes sense!

**Reply by jsjoeio** [reply]
Yeah, that should be fine! Good luck!

**Comment by Jules1111** [comment]
Thank you :) !
